tools/eslint.config.mjs

```diff
@@ -48,6 +48,7 @@ export default [
           allowRuleToRunWithoutStrictNullChecksIKnowWhatIAmDoing: false
         }
       ],
+      '@typescript-eslint/no-var-requires': 'off',
```

Comment:
Was this triggered by `const hooks = require('hooks')` in `tests/all.js` file?

Reply:
This lint should only be applied to typescript files. We should disable it for `js` files only: https://github.com/Shopify/eslint-plugin-shopify/issues/159#issuecomment-485813838

Reply:
That works! 

https://eslint.org/blog/2022/08/new-config-system-part-2/